### PR TITLE
Create basis of the theme color picker

### PR DESCRIPTION
**Pull request by rhetprieto**

### CODE REVIEW

File: .github/workflows/semantic-release.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: CI
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (jadefish) [inline .github/workflows/semantic-release.yml]: I think these should be `main` (or both).

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# 1.0.0 (2021-06-14)
+
+
+### Features
+
+* COPRL presenters v3 ([73807e7](https://github.com/coprl/theme_color_picker_presenter_plugin/commit/73807e71d3c9410ff76437323b3f480784f98b47))
```
Comment (jadefish) [inline CHANGELOG.md]: This commit doesn't exist – was this file copy/pasted?

I think the semantic-release action will take care of these entries, too.

File: LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Russell Edens (russell@voomify.com)
```
Comment (jadefish) [inline LICENSE.txt]: minor, but: this line needs some adjustment.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+And then execute:
+
+    $ bundle
+
+
+## Usage in POMs
+
+Declare the plugin in your pom, `plugin :theme_ color_picker`.
```
Comment (jadefish) [inline README.md]:
```suggestion
Declare the plugin in your pom, `plugin :theme_color_picker`.
```

File: views/assets/css/theme_color_picker.css [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    position: relative;
+    display: inline-block;
+    box-sizing: border-box;
+    padding: .5em 1em;
+    min-height: 2em;
+    border: none;
+    outline: none;
+    overflow: visible;
+    color: white;
+    font-family: sans-serif;
```
Comment (jadefish) [inline views/assets/css/theme_color_picker.css]: Should this match the font stack in COPRL/material design?

File: Gemfile.lock [hunk 6]
```diff
@@ -0,0 +1,20 @@
+PATH
+  remote: .
+  specs:
+    theme_color_picker_presenter_plugin (0.0.1)
```
Comment (jadefish) [inline Gemfile.lock]: I think this needs to (or at least should) match the `ThemeColorPickerPresenterPlugin::VERSION` constant in `lib/theme_color_picker_presenter_plugin/version.rb`.

File: views/components/application/_theme_color_picker.erb [hunk 7]
```diff
@@ -0,0 +1,18 @@
+<div id="<%= comp.id %>"
+     class="v-plugin v-plugin--color-picker v-input mdc-text-field mdc-text-field--outlined v-focusable"
+     style="height: auto; max-width: 100%;"
+     data-plugin-callback="ThemeColorPicker"
+     <% if comp.preview_data %>
+       data-preview-section-id= "<%= comp.preview_data[:section] %>"
+       data-preview-vars='<%= comp.preview_data[:set_vars].presence || [] %>'
```
Comment (jadefish) [inline views/components/application/_theme_color_picker.erb]: This is JSON, right? How/where is it being serialized to JSON, and does it need to be escaped?
```suggestion
       data-preview-vars='<%= h comp.preview_data[:set_vars].presence || [] %>'
```

**Review by jadefish — CHANGES_REQUESTED** [review]

File: views/components/application/_theme_color_picker.erb (hunk 7 above)
Comment (rhetprieto) [reply views/components/application/_theme_color_picker.erb]: It is just an array, I think we should be fine. But it doesn't hurt

**Pull request merged**